config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/pelletier/go-toml/v2"
+)
+
+type SessionName struct {
+	IncludeRootDir bool `toml:"include_root_dir"`
```

Comment:
Let's call this "PrependParentDir" to be more obvious

Reply:
Good idea!